ui/packages/catalog-ui-search/src/main/webapp/js/router.js

```diff
@@ -25,20 +25,20 @@ import React from 'react'
 import { render } from 'react-dom'
 import ExtensionPoints from '../extension-points'
 
-const { routes } = ExtensionPoints
-
-// notfound route needs to come at the end otherwise no other routes will work
-render(<ReactRouter routeDefinitions={routes} />, Application.App.router.$el[0])
+render(
+  <ReactRouter routeDefinitions={ExtensionPoints.routes} />,
+  Application.App.router.$el[0]
```

Comment:
❓ What does this exactly grab?

Reply:
That was already there. It was just reformatted.